packages/table/src/table.tsx

```diff
@@ -512,7 +513,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
      * If no indices are provided, default to using the tallest visible cell from all columns in view.
      */
     public resizeRowsByTallestCell(columnIndices?: number | number[]) {
-        const { locator } = this.state;
+        const locator = this.locator;
```

Comment:
can we just reference `this.locator` directly in this block? I generally prefer top-level things on `this.` to keep the `this.` so it's clear you're using a class member